### PR TITLE
Develop (LIB-346)

### PR DESCRIPTION
**Pull request by shreya-el**
Update capabilities diagram
Add str/int type for repo_key across all functions
Add job logs

### CODE REVIEW

**Review by shilpa-nair-elu — CHANGES_REQUESTED** [review]
could you please confirm on the comment i have made? thanks.

**Comment by shilpa-nair-elu** [comment]
@shreya-el  please add the ticket number in the PR name and also add the link of this PR to the LIB ticket we have in place. helps in tracking later. thanks!

File: polly/Basic.py [hunk 1]
```diff
@@ -22,15 +22,15 @@ def get_all_omixatlas(self, query_api_version="v2", count_by_source=True, count_
...
         query_api_version="v2",
         count_by_source=True,
         count_by_data_type=True):
          
         """
         This function will return you a object that contain information about a given Omixatlas.
         Args:
-              key (str) : repo_id or repo_name.
+            repo_key (str/int): repo_key(repo_name/repo_id)
         Returns:
```
[the remark this answers is not in the document]
Comment (shilpa-nair-elu) [reply polly/Basic.py:34]: resolved. repo id and repo name both supported.

File: polly/Basic.py [hunk 2]
```diff
@@ -48,7 +48,7 @@ def create(
         Args:
              display_name (str): display name of the omixatlas.
              description (str): description of the omixatlas.
-             repo_name (str): repo_name which is used to create index in db.
```
Comment (shilpa-nair-elu) [inline polly/Basic.py:51]: @shreya-el create function in OA takes only repo_name.

**Review by shilpa-nair-elu — CHANGES_REQUESTED** [review]
just one function -  create function in OA takes only repo_name. need to make changes for that. rest looks good.

File: polly/Cohort.py [hunk 3]
```diff
@@ -28,7 +28,7 @@ def create_cohort(
                   local_path (str): local path to instantiate the cohort.
                   cohort_name (str): identifier name for the cohort.
                   description (str): description about the cohort.
-                  repo_key (str): repo_key(repo_name/repo_id) for the omixatlas from where datasets or samples is to be added.
```
[the remark this answers is not in the document]
Comment (shilpa-nair-elu) [reply polly/Cohort.py:31]: resolved.

File: polly/Basic.py (hunk 2 above)
Comment (shreya-el) [reply polly/Basic.py:51]: resolved

**Review by shilpa-nair-elu — APPROVED** [review]
Looks like we have covered all the changes needed. Approving.

**Review by ylakhotia1 — APPROVED** [review]

**Pull request merged**